grafana-image-renderer.yaml

```diff
@@ -0,0 +1,68 @@
+package:
+  name: grafana-image-renderer
+  version: 3.11.6
+  epoch: 0
+  description: A Grafana backend plugin that handles rendering of panels & dashboards to PNGs using headless browser (Chromium/Chrome)
+  copyright:
+    - license: Apache-2.0
+  target-architecture:
+    - x86_64
+  dependencies:
+    runtime:
+      - chromium
+      - dumb-init
+      - nodejs
+
+environment:
+  contents:
+    packages:
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - nodejs
+      - scanelf
+      - ttf-dejavu
+      - vim
+      - yarn
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/grafana/grafana-image-renderer
+      tag: v${{package.version}}
+      expected-commit: 61d808662b646f9fd1ec322ada2cb7acc20f706a
+
+  - runs: |
+      # Bump CVE
+      yarn add cross-spawn@^7.0.5
+      yarn upgrade
```

Comment:
Why we use `yarn upgrade` here instead of `yarn install --production`? https://github.com/grafana/grafana-image-renderer/blob/bf37b95d2af59c1323feaacf19ae28985e35ba84/Dockerfile#L25C5-L25C46

We can simply omit lots of unnecessary dev libs with the `--production` or `--omit=dev` flags.

Reply:
`yarn install --production` doesn't bump the CVE, and `yarn upgrade --production` causes build errors.

Reply:
Thanks Furkan, I didn't realize that you could `yarn install` twice with the prod flag the 2nd time 😅